### ISSUE TITLE
Where are the binary sources?

### ISSUE THREAD

**Issue by 4nd3r50ncr** [crazii/SBEMU#22]
That's fine, but, you see, where are the source codes for the following binaries sbemu.gdt sbemu.grp? Everything would be easier to be able to help, not becoming something personal, but public, if you used software with an open license, since all the code must be open.

sbemu.gdt sbemu.grp?

Thanks.

**Comment by 4nd3r50ncr** [comment]
Friend, don't do that, put the file for direct compilation without conversion, for that, people help each other to improve the code without, if you don't put it, other people will put it.

**Comment by volkertb** [comment]
@4nd3r50ncr Actually, these aren't binary blobs. `sbemu.gdt` and `sbemu.grp` are [RHIDE](https://sourceforge.net/projects/rhide/) project files.

RHIDE is an open source DOS-based IDE, which @crazii used to develop and build this project.

RHIDE comes with a utility program called `GPR2MAK.EXE`, which should be able to convert `sbemu.gpr` to a standard Makefile. Unfortunately, I couldn't get the generated Makefile to build the project, so what I did instead was run the RHIDE build process in a highly slowed down VM, slow enough so I could see and write down the compiler commands that it ran, and used those to write a build script that can cross-compile the project from Linux. A bit tedious, but it worked.

See [this thread](https://github.com/crazii/SBEMU/issues/19) in which I described this.

To be fair, it would indeed have helped, at least for those of us unfamiliar with RHIDE, if this were clarified in the README of the project.

I also have [an outstanding Pull Request](https://github.com/crazii/SBEMU/pull/21) to [automate the build process through a GitHub Actions Workflow](https://github.com/volkertb/SBEMU/blob/set-up-cicd/.github/workflows/01-hello-world.yml). Although I would still prefer to figure out how to use the Makefile generated by `GPR2MAK.EXE`, so the `sbemu.gpr` remains a "single source of truth" in terms of building the project, and the "reverse-engineered" cross-compilation script doesn't have to be maintained manually beside it. Help with this would definitely be welcome! :slightly_smiling_face:
